### ISSUE TITLE
"heatOnly": true - ignores auto mode

### ISSUE THREAD

**Issue by duculete** [PJCzx/homebridge-thermostat#54]
if headonly = true, maybe i want to have the following:

off - boiler is off permanently
on - boiler is allways on
auto - based on temperature threshold if currTemp < targetTemp - start heating, if currTemp > targetTemp - stop heating

update code:

if (this.heatOnly) {
                        this.service.getCharacteristic(Characteristic.TargetHeatingCoolingState)
                                .setProps({
                                        minValue: 0,
                                        maxValue: 3, < 0...3
                                        validValues: [0,1,3] <<< add 3, ignore 2 (cooling)
                                });

**Comment by PJCzx** [comment]
Yeah great :) Do you mind to make a pull request in order to add a "working code" directly ? I do not use the plugin anymore I'm just taking care of the repo by now...
